integration-tests/src/test/resources/apps/httpsessionreptestapp/CounterServlet.java

```diff
@@ -0,0 +1,102 @@
+package apps.httpsessionreptestapp;
+
+import java.io.*;
+import javax.servlet.*;
+import javax.servlet.http.*;
+import weblogic.servlet.internal.session.RSID;
+import weblogic.servlet.internal.MembershipControllerImpl;
+import weblogic.servlet.spi.WebServerRegistry;
+import java.util.Enumeration;
+
+public class CounterServlet extends HttpServlet {    
```

Comment:
javadoc

Reply:
added

/**
 * Simple HTTP servlet class for a client to 
 * 1. query weblogic primary and secondary server
 * 2. set or get a count number 
 */

Reply:
added

/**
 * Simple HTTP servlet class for a client to 
 * 1. query weblogic primary and secondary server
 * 2. set or get a count number 
 */